integration-tests/src/test/java/org/apache/fineract/integrationtests/datatable/DatatableUniqueAndIndexColumnTest.java

```diff
@@ -176,6 +176,7 @@ public void testDatableModificationWithUniqueAndIndexedColumns() {
         changeColumns.setName(column1Name);
         String newColumnName = column1Name + "new";
         changeColumns.setNewName(newColumnName);
+        changeColumns.setUnique(false);
```

Comment:
I am adding a note on my PR because I would like this change to be approved. 
This field was requested to be unique: true, indexed: true already on create. So sending an update request with the indexed flag to be true can not set the unique flag to false. I think this was a bug, but please confirm.

Reply:
Sounds like one...

Reply:
Exactly, that was the problem, not sending something on update means no change and not set to default value. If the field is mandatory, then it was already set on insert.